fontforge/fvcomposite.c

```diff
@@ -50,20 +50,17 @@ int CharCenterHighest = 1;
 #define BottomAccent	0x300
 #define TopAccent	0x345
 
-/* for accents between 0x300 and 345 these are some synonyms */
-/* type1 wants accented chars built with accents in the 0x2c? range */
-/*  except for grave and acute which live in iso8859-1 range */
 /*  this table is ordered on a best try basis */
```

Comment:
Now, as reviewers, I hope the folks on this thread can help out with the other issue posed by this push: what this table should look like. 

My question is: Shouldn't every entry either start with 0 or with the corresponding 0x3?? value? (Or maybe just the latter, for clarity's sake?) @ctrlcctrlv has been conservative in just reordering the entries and not adding anything, but aren't "hook above" and "reversed comma" even more problematic?

Reply:
@skef, do all of the accents in the table have combining marks?

Reply:
I believe it does by definition. It's indexed like `apt = accents[uni-BottomAccent]`, so what the table represents is the set of combining characters between 0x300 and 0x345.  There aren't any "empties" in that range: https://vazor.com/unicode/b0300.html

Reply:
I would concur with your suggestion then. George may know something that we don't here, but it seems to me that we ought to be consistent across the whole range. @ctrlcctrlv?

Reply:
@skef You're right, I was overly conservative, and I've remedied that in my latest commit to this branch.